cime_config/user_nl_blom_outvol_reduced

```diff
@@ -0,0 +1,534 @@
+  FLX_DUST2000      = 2, 2
+  FLX_DUST4000      = 2, 2
+  FLX_DUST_BOT      = 2, 2
+  FLX_NDEPNHX       = 2, 2
+  FLX_NDEPNOY       = 2, 2
+  FLX_OALK          = 0, 0
+  FLX_SEDIFFAL      = 0, 2
+  FLX_SEDIFFIC      = 0, 2
+  FLX_SEDIFFN2      = 0, 2
+  FLX_SEDIFFN2O     = 0, 2
+  FLX_SEDIFFNH4     = 0, 2
+  FLX_SEDIFFNO2     = 0, 2
+  FLX_SEDIFFNO3     = 0, 2
+  FLX_SEDIFFOX      = 0, 2
+  FLX_SEDIFFPH      = 0, 2
+  FLX_SEDIFFSI      = 0, 2
+  FLX_BURSSO12      = 0, 2
+  FLX_BURSSSC12     = 0, 2
+  FLX_BURSSSSIL     = 0, 2
+  FLX_BURSSSTER     = 0, 2
+
+! All layer-output disabled for reduced outpu setting
+
+  LYR_AGG_AVDC      = 0, 0
+  LYR_AGG_AVDP      = 0, 0
+  LYR_AGG_AVRHOP    = 0, 0
+  LYR_AGG_B         = 0, 0
+  LYR_AGG_DF        = 0, 0
+  LYR_AGG_DMAX      = 0, 0
+  LYR_AGG_STICK     = 0, 0
+  LYR_AGG_STICKF    = 0, 0
+  LYR_AGG_VPOR      = 0, 0
+  LYR_AGG_VRHOF     = 0, 0
+  LYR_AGG_WS        = 0, 0
+  LYR_ANH4          = 0, 0
+  LYR_ANMX_N2_PROD  = 0, 0
+  LYR_ANMX_OM_PROD  = 0, 0
+  LYR_ANO2          = 0, 0
+  LYR_ANO3          = 0, 0
+  LYR_DENIT_N2O     = 0, 0
+  LYR_DENIT_NO2     = 0, 0
+  LYR_DENIT_NO3     = 0, 0
+  LYR_DNRA_NO2      = 0, 0
+  LYR_DYNVIS        = 0, 0
+  LYR_NITR_N2O_PROD = 0, 0
+  LYR_NITR_NH4      = 0, 0
+  LYR_NITR_NH4_OM   = 0, 0
+  LYR_NITR_NO2      = 0, 0
+  LYR_NITR_NO2_OM   = 0, 0
+  LYR_PHOSY_NH4     = 0, 0
+  LYR_PHOSY_NO3     = 0, 0
+  LYR_REMIN_AEROB   = 0, 0
+  LYR_REMIN_SULF    = 0, 0
+  LYR_PHYTO         = 0, 0
+  LYR_GRAZER        = 0, 0
+  LYR_DOC           = 0, 0
+  LYR_PHOSY         = 0, 0
+  LYR_PHOSPH        = 0, 0
+  LYR_OXYGEN        = 0, 0
+  LYR_IRON          = 0, 0
+  LYR_ALKALI        = 0, 0
+  LYR_SILICA        = 0, 0
+  LYR_DIC           = 0, 0
+  LYR_POC           = 0, 0
+  LYR_CALC          = 0, 0
+  LYR_OPAL          = 0, 0
+  LYR_CO3           = 0, 0
+  LYR_N2O           = 0, 0
+  LYR_PH            = 0, 0
+  LYR_OMEGAC        = 0, 0
+  LYR_OMEGAA        = 0, 0
+  LYR_PREFO2        = 0, 0
+  LYR_O2SAT         = 0, 0
+  LYR_PREFPO4       = 0, 0
+  LYR_PREFALK       = 0, 0
+  LYR_PREFDIC       = 0, 0
+  LYR_PREFSILICA    = 0, 0
+  LYR_DICSAT        = 0, 0
+  LYR_NATDIC        = 0, 0
+  LYR_NATALKALI     = 0, 0
+  LYR_NATCO3        = 0, 0
+  LYR_NATCALC       = 0, 0
+  LYR_NATPH         = 0, 0
+  LYR_NATOMEGAC     = 0, 0
+  LYR_NATOMEGAA     = 0, 0
+  LYR_DIC13         = 0, 0
+  LYR_DIC14         = 0, 0
+  LYR_D13C          = 0, 0
+  LYR_D14C          = 0, 0
+  LYR_BIGD14C       = 0, 0
+  LYR_POC13         = 0, 0
+  LYR_DOC13         = 0, 0
+  LYR_CALC13        = 0, 0
+  LYR_PHYTO13       = 0, 0
+  LYR_GRAZER13      = 0, 0
+  LYR_CFC11         = 0, 0
+  LYR_CFC12         = 0, 0
+  LYR_SF6           = 0, 0
+  LYR_NOS           = 0, 0
+  LYR_WPHY          = 0, 0
+  LYR_WNOS          = 0, 0
+  LYR_EPS           = 0, 0
+  LYR_ASIZE         = 0, 0
+  LYR_BROMO         = 0, 0
+  LYR_SHELFAGE      = 0, 0
+  LYR_DP@diabgc     = 0, 0
+
+! Save level-fields only annually
+
```

Comment:
I am wondering, how much info we would need for the sinking. Maybe it would reasonable during spinup to at least write out the LVL_AGG_WS - to enable having a rough idea on e.g. remineralization length scales? - for production runs, I would clearly opt for some more information later.